docs/api-docs/storefront/scripts-overview.md

```diff
@@ -2,31 +2,26 @@
 <div class="otp" id="no-index">
 	<h3> On This Page </h3>
 	<ul>
-    		<li><a href="#scripts_script-manager-partner-guidelines">Script Manager API Partner Guidelines</a></li>
+    	<li><a href="#scripts_script-manager-partner-guidelines">Script manager API partner guidelines</a></li>
         <li><a href="#script_prerequisites">Prerequisites</a></li>
-        <li><a href="#scripts_upgrades-installation">Upgrades and Installation</a></li>
-        <li><a href="#scripts_fixing-missing-scripts">Fixing Missing Scripts</a></li>
+        <li><a href="#scripts_upgrades-installation">Upgrades and installation</a></li>
+        <li><a href="#scripts_fixing-missing-scripts">Fixing missing scripts</a></li>
         <li><a href="#scripts_scripts-notes">Notes</a></li>
-        <li><a href="#scripts_scripts-visibility">Visibility Locations </a></li>
+        <li><a href="#scripts_scripts-visibility">Visibility locations </a></li>
 	</ul>
 </div>
 
-The BigCommerce Scripts API gives developers the ability to inject scripts into a store's template files programmatically. This means that app and integrations can insert scripts into a user’s storefront without requiring the user to manually paste a snippet of code into their control panel. There are many use cases for this powerful API, for example:
+The BigCommerce Scripts API gives developers the ability to inject scripts into a store's template files programmatically. This ability means that app and integrations can insert scripts into a user’s storefront without requiring the user to paste a snippet of code into their control panel manually. There are many use cases for this powerful API, for example:
 * inserting analytics scripts
 * inserting single-click app scripts
 * inserting live chat and support plugins
 * inserting theme extensions or connector apps
 

```

Comment:
Remove the leading "inserting" in the list items above to avoid redundancy. 
Bullet points should also be followed by capital letters